test/test_examples.sh

```diff
@@ -67,7 +77,7 @@ for ex in $EXAMPLES; do
 	rm -rf $BLD_DIR
 	mkdir -p $BLD_DIR
 	cd $BLD_DIR
-	CMAKE_PREFIX_PATH="$INSTALL_DIR" cmake $SRC_DIR
+	CMAKE_PREFIX_PATH="${INSTALL_DIR}/${CMAKE_INSTALL_PREFIX}" cmake $SRC_DIR
```

Comment:
we could just install to the `<install dir>/<example_name>_standalone` or something

Reply:
Please read carefully https://github.com/oneapi-src/unified-memory-framework/pull/679#discussion_r1717288909 (August 14th):

The command:
```
$ make DESTDIR=/package/stage install
```
installs all files to the `${DESTDIR}/${CMAKE_INSTALL_PREFIX}` directory.